randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -101,17 +103,13 @@ Extra - asset_id: 68
   * Layers: default
   * Normal Door to Green Brinstar Main Shaft/Door to Brinstar Pre-Map Room
   > Door to Brinstar Map Room
-      All of the following:
-          Morph Ball
-          Morph Ball Bombs or Power Bombs
+      Can Lay Any Bombs
 
 > Door to Brinstar Map Room; Heals? False
   * Layers: default
   * Normal Door to Brinstar Map Room/Door to Brinstar Pre-Map Room
   > Door to Green Brinstar Main Shaft
-      All of the following:
-          Morph Ball
-          Morph Ball Bombs or Power Bombs
+      Can Lay Any Bombs
```

Comment:
Yes, but they're beginner enemies that all die to Power Beam. Should an event be placed anyway?

Reply:
Not sure. I remembered this, because VARIA goes out of its way to remove the locked door due to players getting softlocked otherwise. Maybe a comment is enough? Maybe keep it as is?

Reply:
I think the issue is that one of the enemies is on the other side of a bomb block, and if you don't have bombs or wave you can't kill it. I'm not sure where the requirement should be put. If you put it in the room it won't stop the player from walking in the room. If you put it on the connection to the room then its wrong because you don't need bombs to get in. Not sure what to do.

Reply:
id say we just put a comment into one of the connections. it may have some future relevancy